BitwardenShared/Core/Vault/Services/SyncService.swift

```diff
@@ -90,6 +92,14 @@ protocol SyncServiceDelegate: AnyObject {
     func setMasterPassword(orgIdentifier: String) async
 }
 
+// MARK: - SyncService extension
+
+extension SyncService {
+    func needsSync(for userId: String) async throws -> Bool {
+        try await needsSync(for: userId, onlyCheckLocalData: false)
```

Comment:
🤔 It seems like we don't use this, should we keep this?

Reply:
You're totally right, removed; thanks for catching this!